### ISSUE TITLE
`no-commented-out-code`: Unexpectedly triggered for some magic Webpack comments

### ISSUE THREAD

**Issue by TravelingTice** [cartant/eslint-plugin-etc#31]
Hi there! We're having a project where we're using code splitting and noticed that the `no-commented-out-code` was triggered for some [Webpack Magic comments](https://webpack.js.org/api/module-methods/#magic-comments) we can use in these code split import functions. For example `webpackChunkName`:

```js
const Component = lazy(() => import(/* webpackChunkName: 'my_component' */ './component'))
```

For some magic comments it didn't trigger, for example `/* webpackMode: true */`, `/* webpackPreload: true */`, `/* webpackPrefetch: true */`.

It looks like it only triggers for magic comments with a string, array or regex as a value for the webpack option.

**Comment by cartant** [comment]
This should be fixed in 1.5.1. I've tweaked the implementation so that instead of specifically looking for comments that could parsed as labelled statements, it just considers lone, labelled statements to be non-code.

**Comment by TravelingTice** [comment]
Awesome! It works great! Thanks for the fast response 🤩
